app/models/miq_provision_virt_workflow.rb

```diff
@@ -620,12 +612,6 @@ def get_source_and_targets(refresh = false)
     result
   end
 
-  def source_valid?
```

Comment:
This method is not being used and can be deleted. 
[template_valid?](https://github.com/ManageIQ/manageiq/blob/bb302b728a49d27aa871f81f5c65139725f97b79/app/models/service_template.rb#L213) will be used to validate the service template as in pending PR [#4415](https://github.com/ManageIQ/manageiq/pull/4415/files#diff-5ae75ae3d639d820a634283662b2f3efR38).